MediaBrowser.Model/Lyrics/LyricLine.cs

```diff
@@ -25,4 +29,9 @@ public LyricLine(string text, long? start = null)
     /// Gets the start time in ticks.
     /// </summary>
     public long? Start { get; }
+
+    /// <summary>
+    /// Gets the Enhanced LRC timestamps for the song.
```

Comment:
The lyric data structure is format independent and should **not** be made for one specific format.